tests/tt_metal/tt_metal/perf_microbenchmark/3_pcie_transfer/test_pull_from_pcie.cpp

```diff
@@ -68,7 +68,7 @@ void* align(void* ptr, std::size_t max_alignment) {
     // ex. if the current ptr here is 16, but we specified an alignment of 8,
     // then this is both 8 and 16 byte aligned, so we offset again by our
     // specified alignment to make the max alignment what was specified
-    aligned = aligned & (max_alignment << 1 - 1) ? aligned : aligned + max_alignment;
+    aligned = aligned & (max_alignment << (1 - 1)) ? aligned : aligned + max_alignment;
```

Comment:
```
[build] /home/afuller/src/metal-1/tests/tt_metal/tt_metal/perf_microbenchmark/3_pcie_transfer/test_pull_from_pcie.cpp:71:45: error: operator '<<' has lower precedence than '-'; '-' will be evaluated first [-Werror,-Wshift-op-parentheses]
[build]    71 |     aligned = aligned & (max_alignment << 1 - 1) ? aligned : aligned + max_alignment;
[build]       |                                        ~~ ~~^~~
[build] /home/afuller/src/metal-1/tests/tt_metal/tt_metal/perf_microbenchmark/3_pcie_transfer/test_pull_from_pcie.cpp:71:45: note: place parentheses around the '-' expression to silence this warning
[build]    71 |     aligned = aligned & (max_alignment << 1 - 1) ? aligned : aligned + max_alignment;
[build]       |                                             ^  
[build]       |                                           (    )
[build] 1 error generated.
```

I suspect what was intended here is:
```suggestion
    aligned = aligned & ((max_alignment << 1) - 1) ? aligned : aligned + max_alignment;
```
Yes?  Why aren't tests failing as a result?

Reply:
Yes, should be `((max_alignment << 1) - 1)`. This wouldn't cause any functional issues, it would just affect performance if there is a perf difference in alignment. I think the conclusion that there wasn't any perf difference which is why there probably wasn't any issues? Would need to review the test again to confirm.